modular_skyrat/modules/alternative_job_titles/code/alt_job_titles.dm

```diff
@@ -19,17 +19,32 @@
 /datum/job/assistant
 	alt_titles = list(
 		"Assistant",
+		"Ambassador",
 		"Artist",
+		"Actor",
 		"Businessman",
 		"Businesswoman",
+		"Butler",
+		"Colonist",
+		"Contractor",
 		"Civilian",
+		"Duke",
```

Comment:
An assistant with the title duke seems very out of place on a space station.

Reply:
I thought king was too high, I wanted something for minor noble gimmicks.